interface/main/finder/dynamic_finder.php

```diff
@@ -50,7 +50,7 @@
     $header .= text($title);
     $header .= "</th>\n";
     $header0 .= "   <td ><input type='text' size='20' ";
-    $header0 .= "value='' class='search_init' placeholder='" . xla("Search by") . " " . $title1 . "'/></td>\n";
+    $header0 .= "value='' class='form-control' placeholder='" . xla("Search by") . " " . $title1 . "'/></td>\n";
```

Comment:
Does this class go to anything?

Reply:
Yes yes yes, please ensure jquery or js getbyclass is not using somewhere!

Reply:
I get in-depth and found it may relative with datatables, but I am not sure how to trigger it.
So I add the `search_init` it again.

https://legacy.datatables.net/examples/api/multi_filter.html